scipy-notebook/test/test_extensions.py

```diff
@@ -7,6 +7,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
+@pytest.mark.skip(reason="Not yet compliant with JupyterLab 3")
```

Comment:
All of three extensions do not work or only some of them?
It might be better to comment the ones which do not work, rather than all three of them.

Reply:
In fact like I said in the limitations paragraph, only `bokeh` is not available and do not work. 
For the others it's the command `jupyter labextension check $extension` that returns an error code whatever the extension. I guess it's a bug in the `check` subcommand because `list` returns OK. However I do not want to recode the test. This is the reason why everything is skipped even if you're right if it had been coded with the `list` subcommand only `bokeh` should have been skipped.

```bash
jupyter labextension list
# JupyterLab v3.0.5
# /opt/conda/share/jupyter/labextensions
#         jupyter-matplotlib v0.8.2 enabled OK
#         @jupyter-widgets/jupyterlab-manager v3.0.0 enabled OK (python, jupyterlab_widgets)
```

Reply:
Ok, thanks for the explanation.